pika/heartbeat.py

```diff
@@ -129,7 +124,7 @@ def _close_connection(self):
         """Close the connection with the AMQP Connection-Forced value."""
         LOGGER.info('Connection is idle, %i stale byte intervals',
                     self._idle_byte_intervals)
-        duration = self._max_idle_count * self._interval
+        duration = HeartbeatChecker._MAX_IDLE_COUNT * self._timeout
```

Comment:
Don't need this since will be using `self._check_interval` in _adapter_add_timeout() below.